ExtraEnemyCustomization/Patches/TentacleAttackPatches.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Enemies;
+using HarmonyLib;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using EEC.Managers;
+using System.Linq;
+using EEC.EnemyCustomizations.Models;
+using EEC.Configs.Customizations;
+using EEC.API;
+
+namespace EEC.Patches
+{
+    [HarmonyPatch(typeof(ES_EnemyAttackBase))]
+    internal class TentacleAttackPatches // Need to know where to move this
```

Comment:
`Inject_TYPENAME_METHODNAME` or `Inject_TYPENAME` or `Inject_WHATITDOES` is naming convention for Harmony Injection classes, Also needs to be static